src/object/cli_shard.c

```diff
@@ -1353,8 +1403,10 @@ dc_enumerate_copy_csum(d_iov_t *dst, const d_iov_t *src)
 		       min(dst->iov_buf_len,
 			   src->iov_len));
 		dst->iov_len = src->iov_len;
-		if (dst->iov_len > dst->iov_buf_len)
+		if (dst->iov_len > dst->iov_buf_len) {
+			D_INFO("Checksum buffer truncated");
```

Comment:
I presume this doesn't happen often?   Or it should probably be a debug message.